### ISSUE TITLE
Success: null

### ISSUE THREAD

**Issue by WaterHyacinthInNANHU** [waylaidwanderer/node-chatgpt-api#470]
**Describe the bug**
I'm using BingAIClient. After several successful queries, the API server returns error code `503` with the error message: `{"error":"null"}`. Please help~

**Screenshots**
![image](https://github.com/waylaidwanderer/node-chatgpt-api/assets/50191547/3d8a12db-1e76-45f5-ac68-9cd5a9e8dbbf)

**Node.js version (please complete the following information):**
 - OS: Linux 4090 5.15.0-82-generic 20.04.1-Ubuntu
 - Node v20.5.1

**Package version (please complete the following information):**
 -  @waylaidwanderer/chatgpt-api@1.37.3

**Comment by vaibhavard** [comment]
Same Issue

**Comment by jegue** [comment]
yeah same here! If using the SRCHLANG cookie.

**Comment by vaibhavard** [comment]
I use _U cookie .. Whats srchlang cookie?

**Comment by Richard-Weiss** [comment]
@WaterHyacinthInNANHU
It seems to be a problem on Microsoft's side. Sometimes a conversationSignature won't be returned when sending the conversation POST.
I don't experience the problem anymore after getting new optionSets and SliceIds from a vanilla request, however there still seems to be some problems on Microsoft's side, mainly InternalServiceErrors for normal POST requests.
So you might just see the error "Error processing request." displayed sometimes.
The issue isn't really related to the repo code itself, but we should leave it open until the errors disappear, so nobody creates a new issue.

**Comment by WaterHyacinthInNANHU** [comment]
> @WaterHyacinthInNANHU It seems to be a problem on Microsoft's side. Sometimes a conversationSignature won't be returned when sending the conversation POST. I don't experience the problem anymore after getting new optionSets and SliceIds from a vanilla request, however there still seems to be some problems on Microsoft's side, mainly InternalServiceErrors for normal POST requests. So you might just see the error "Error processing request." displayed sometimes. The issue isn't really related to the repo code itself, but we should leave it open until the errors disappear, so nobody creates a new issue.

It is true that starting a new conversation solves this. Thanks for your timely help!

**Comment by Richard-Weiss** [comment]
Yeah, it seems to have "solved itself" i.e. they fixed it on their end.

**Comment by michael-ts** [comment]
I am getting this error out of the gate. I have not been able to get any successful responses, ever.
One thing that is different than the above error is that once I paste my cookies into settings.txt the error message changes from "|BingProdUnAuthenticatedUsers|" to "|BingProd|".
